Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -7214,6 +7214,21 @@ enum class WebCore::ContextMenuItemType : uint8_t {
     Submenu,
 };
 
+#if PLATFORM(COCOA)
+header: <WebCore/TextAttachmentForSerialization.h>
+[CustomHeader] struct WebCore::TextAttachmentMissingImage {
+}
+
+[CustomHeader] struct WebCore::TextAttachmentFileWrapper {
+#if !PLATFORM(IOS_FAMILY)
+    bool ignoresOrientation;
+#endif
+    std::optional<String> preferredFilename;
```

Comment:
I really don't think these need to be std::optional.  Just check if the String is null.  Same with data, accessibilityLabel.